openwisp_monitoring/monitoring/migrations/0015_alertsettings_is_active.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.0.3 on 2020-07-07 09:06
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('monitoring', '0014_merge_traffic_metrics_and_fill_metric_config'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
```

Comment:
why are you using RunSQL?? I don't understand why we can't use a regular migration like all the other ones? Please explain.

Reply:
I stated it above https://github.com/openwisp/openwisp-monitoring/pull/150#issuecomment-654786288 (last para), maybe I should use inline comments :sweat_smile: